java/src/main/java/com/google/oak/transport/EvidenceProvider.java

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.transport;
+
+import com.google.oak.util.Result;
+import oak.session.noninteractive.v1.AttestationBundle;
+
+/** * An interface for providing an enclave evidence. */
```

Comment:
Is this going to be called evidence or attestation bundle? I think it should be consistent

Reply:
I don't really like the name `AttestationBundle`

We either can create a new name for Evidence+Endorsement, or always separate them as different proto messages (that are send to the clients together)

Reply:
I'll update it in a separate PR (since naming also affects Rust)